Replace `pari` with new `ac.Options` struct (#140)
MIT-LAE/TASOPT.jl: docs/src/balance/balance.md

This module provides functions to size the tail surfaces and balance the aircraft to achieve pitch trim throughout flight when called by [`wsize()`](@ref TASOPT.wsize).

- [`balance()`](@ref TASOPT.balance) makes adjustments as described below to achieve pitch trim. It calculates the aircraft's resulting **center of gravity (`xCG`)**, **center of pressure (`xCP`)**, and **neutral point (`xNP`)** at a specific flight point. To meet the pitch trim requirement, the routine adjusts one of (i) the horizontal tail's lift coefficient, (ii) its area, or (iii) the axial location of the wing box. In *almost all* the use cases, only option (i) adjusting the horizontal tail's lift coefficient is relevant for trim calculations.

!!! warning "Some `balance()` options change the aircraft design"
    Exercise caution if options (ii) or (iii) are being used as these options *alter the design of the aircraft* (i.e., they should not be used in off-design cases).

> - [`size_htail()`](@ref TASOPT.size_htail) performs a more involved stability analysis: it determines the **horizontal tail area (`Sh`)** and **wing box position (`xwbox`)**, solving for pitch trim and static stability across all flight conditions.

Reviewer: ```suggestion
- [`size_htail()`](@ref TASOPT.size_htail) simultaneously sizes **horizontal tail area (`Sh`)** and positions the **wing box (`xwbox`)**, by solving for pitch trim and static stability across all flight conditions.
```
A little much to call this an involved stability analysis.

Author: the siren's song of adverbs
agreed
